PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -157,6 +160,21 @@ class PsfPowershellScriptRunner
 			std::wstring exitCodeStr = std::to_wstring(exitCode);
 			MessageBoxEx(NULL, (script.scriptPath + std::wstring(L" execution failed with Exit Code ") + exitCodeStr + std::wstring(L". To fix this, please run ") + script.scriptPath + std::wstring(L" standalone in a PowerShell window and confirm that the value of $LASTEXITCODE is 0 (Success) before using it with PSF.")).c_str(), L"Package Support Framework", MB_OK | MB_ICONWARNING, 0);
 		}
+
+		TraceLoggingWrite(
+			g_Log_ETW_ComponentProvider,
+			"ScriptInformation",
+			TraceLoggingWideString((&script == &(this->m_startingScriptInformation)) ? L"StartScript" : L"EndScript", "ScriptType"),

```

Comment:
I don't see any documentation about adding new TraceLogging calls to new fixups.  I suggest to add documentation on where, and how, to add traces.